docs/schemas/BDAAverager.yml

```diff
@@ -0,0 +1,42 @@
+description: >-
+  Average data in time and/or frequency, depending on baseline lenghts. For
+  shorter baselines, the averaging factor is higher than for longer baselines `.`
+inputs:
+  step_name:
+    type: string
+    doc: unique name for the step `.`
+    default: bdaaverage
+  type:
+    type: string
+    doc: Case-insensitive step type; must be 'bdaaverager' `.`
+    default: bdaaverager
+  timethresholdlength:
+    type: double
+    doc: >-
+      The baseline length (in meters) after which the averager does not perform
+      any averaging over time. The time averaging factor for a baseline becomes
+      timethresholdlength / baseline length, rounded down.
+      The maxinterval setting may limit the time averaging factor `.`
+    default: 0.0
+  freqthresholdlength:
+    type: int
+    doc: >-
+      The baseline length (in meters) after which the averager does not perform
+      any averaging over frequencies. The number of channels for a baseline
+      becomes baseline length / freqthresholdlength * original number of
+      channels, rounded up.
+      The minchannels setting specifies the minimum number of channels.
+      The maximum numbef of channels is the original number of channels `.`
+    default: 0.0
+  maxinterval:
+    type: int
+    doc: >-
+      The maximum interval duration (in seconds), when averaging over time.
```

Comment:
What happens if this is not a multiple of the integration time?

Reply:
Effectively the maximum interval duration is rounded down to the nearest multiple of the integration time. I'll add a comment.